torch/_prims/debug_prims.py

```diff
@@ -29,8 +29,8 @@ def register_debug_prims():
     @custom_op("debugprims::load_tensor")
     def load_tensor(
         name: str,
-        size: Tuple[int, ...],
-        stride: Tuple[int, ...],
```

Comment:
What happens if you specify `List[int]`, or `Tuple[int, ...]`. Is there a loud error? Are docs for "stuff you should know when writing a custom op schema" somewhere on the roadmap?

Reply:
> Is there a loud error?

There is a loud error that tells you what the accepted types are.

> Are docs for "stuff you should know when writing a custom op schema" somewhere on the roadmap?

Yeah, all of this needs a documentation page somewhere. I began one as part of the PT2 hackathon: https://docs.google.com/document/d/16YsyKY5uil_HF5mwa6UOtw0AxEHJkZp06NZcSJ3GyJs/edit#heading=h.deyt6pleeqxt , will finish after we figure out autograd support and if there should be special handling for triton